source/fundamentals/connection.txt

```diff
@@ -245,6 +245,16 @@ that Context deadline for the operation. If the context does not have a
 deadline, the driver derives a new Context from the given Context using
 the ``Timeout`` option set on the ``Client``.
 
+.. note:: Retryability
+
+   This version of the {+driver-short+} enables retryable reads and
```

Comment:
Issue:
This text will become inaccurate when releasing newer driver versions of the docs unless changed and changing details like this is generally outside of our normal version update process.

Suggestion:
Use "Starting in (version)...", or the versionadded directive.